### ISSUE TITLE
Show number of planned uploads/uploads left

### ISSUE THREAD

**Issue by mindmeshlink** [maralorn/nix-output-monitor#120]
When pushing paths to remote machines, I sometimes miss the line:
"copying 2321 paths..." as it gets scrolled up by a sea of individual path copies. It'd be lovely to have `nom` read that line, and indicate the total number of copies it's going to need to do/remaining amount, even if just per single machine.

**Comment by maralorn** [comment]
Yeah, a very sensible request. I vaguely remember initially trying to do this, which was impossible for some reason. I think the "copying 2321 paths" message often lacks too much context to tell from where to where copies are planned.
